src/components/block-list/block-list.js

```diff
@@ -310,13 +310,19 @@ export class BlockList extends LitElement {
                 itemName = 'Unnamed Item';
               }
 
+              // Check if the copy button should be disabled
+              const disableCopy = sectionLibraryMetadata.disablecopy
```

Comment:
in the readme this is camelcase, not sure if it gets normalized somewhere

Reply:
@maxakuru Thanks, yeah it does get normalized. I camel cased it in the docs for readability but in reality it doesn't have to be.